bin/pullTargets.js

```diff
@@ -0,0 +1,15 @@
+require("dotenv").config();
```

Comment:
in insight, the bulk of the code in bin/config should have been in the specific  files... #torefactoroneday